tests/gateway/test_integration.py

```diff
@@ -352,3 +381,91 @@ def _mock_request_session(max_retries, backoff_factor, retry_codes):
         requests.exceptions.HTTPError, match="Unprocessable Entity for"
     ):
         query(route=route.name, data=data)
+
+
+def test_mlflow_chat(gateway):
```

Comment:
Can we test this without mock?

Reply:
I am neutral on the issue. On the other hand, mock test can speedup CI running.

Reply:
Correct test results are better than CI taking longer.

Reply:
Added! :)